sdk/identity/identity/README.md

```diff
@@ -1,3 +1,99 @@
-# Azure SDK Identity Library
+# Azure SDK Identity Library for C
```

Comment:
We're going to get rid of this directory and put "credentials" (which used to be called identity) in the core directory.

Reply:
Is the package going away as well?  Are all languages doing this?

Reply:
This readme to be removed when the Identity package is folded into core